config/crd/v2alpha1/bases/velero.io_datauploads.yaml

```diff
@@ -79,7 +79,7 @@ spec:
                 nullable: true
                 properties:
                   snapshotClass:
-                    description: StorageClass is the name of the snapshot class that
+                    description: SnapshotClass is the name of the snapshot class that
```

Comment:
Please run make update and commit all the changes, i.e., crds.go

Reply:
Thanks .
I used to run it.
But it will update a lot files.
I will rebased later.

Reply:
@Lyndon-Li 
Updated.

Reply:
This still doesn't look correct:

- v1/crds/crds.go should not affected
- v2alpha1/crds/crds.go should not be that long

I suggest you rollback to the latest main code, do the changes again and then run `make update`

Reply:
<img width="1167" alt="image" src="https://github.com/vmware-tanzu/velero/assets/12080746/00a46bc5-42f9-4c81-90d3-f3f17d2a4c36">

Reply:
@Lyndon-Li 
Thanks for quick review.
I actually follow your suggestion , and run it.
The pic above shows that the result is both 
``` v1/crds/crds.go``` and  ```v2alpha1/crds/crds.go ``` had affected.

Is that a possible , the crd had been modified before, but not execute the 'make update'?

Reply:
Yes, you are right. It should be due to this PR https://github.com/vmware-tanzu/velero/pull/6415.
Then I think we can fix this problem in this PR.

Thanks.